README.adoc

```diff
@@ -267,45 +266,46 @@ The methods that you wrote in the [hotspot=Consumer file-0]`Consumer` class coul
 party service, you should separate your `GET`/`POST` requests from your data consumption.
 
 // =================================================================================================
-// Building and running the application
+// Running the application
 // =================================================================================================
 
-[role=command]
-include::{common-includes}/mvnbuild.adoc[]
+== Running the application
 
-When the server is running, you can find your service at http://localhost:9080/artists[http://localhost:9080/artists^]
+The Open Liberty server was started in development mode at the beginning of the guide and all the changes were automatically picked up.
 
-[role=command]
-include::{common-includes}/mvncompile.adoc[]
+You can find your service at http://localhost:9080/artists[http://localhost:9080/artists^].
+
+You can now access the endpoints at http://localhost:9080/artists/total[http://localhost:9080/artists/total^] to see the total number of artists,
+and at `\http://localhost:9080/artists/total/<artist>` to see a particular artist's total number of albums. 
```

Comment:
Change
"You can now access the endpoints at http://localhost:9080/artists/total[http://localhost:9080/artists/total^] to see the total number of artists, and at `\http://localhost:9080/artists/total/<artist>` to see a particular artist's total number of albums. "
To
"Now, you can access the endpoint at http://localhost:9080/artists/total[http://localhost:9080/artists/total^] to see the total number of artists, and you can access the endpoint at `\http://localhost:9080/artists/total/<artist>` to see a particular artist's total number of albums. "